lib/telepoison.ex

```diff
@@ -16,6 +16,8 @@ defmodule Telepoison do
   alias HTTPoison.Request
   alias OpenTelemetry.Tracer
 
+  @default_attributes [:env, :service, :"service.name"]
```

Comment:
I recently discovered that this could be written as `~w(env service service.name)a`, not really sure if I prefer it though

Reply:
Cool, cheers - removed for now based on the other feedback!